app/js/components/tabs/tabcomponents/drugOrderComponent.jsx

```diff
@@ -38,7 +38,7 @@ class DrugOrderComponent extends Component {
             });
         this.props.fetchData('concept?name=REASON%20ORDER%20STOPPED')
             .then(conceptReason => {
-                this.props.fetchData(`concept/${conceptReason.results[0].uuid}`)
+                conceptReason.results && this.props.fetchData(`concept/${conceptReason.results[0].uuid}`)
```

Comment:
@andela-jomadoye i just wanna make sure that you did see these automated reviews. Did you?